pkg/config/config.go

```diff
@@ -65,19 +66,27 @@ func (c *Config) RegisterFlags(f *flag.FlagSet) {
 }
 
 // LoadFile reads a file and passes the contents to Load
-func LoadFile(filename string, c *Config) error {
+func LoadFile(filename string, expandEnvVars bool, c *Config) error {
 	buf, err := ioutil.ReadFile(filename)
 	if err != nil {
 		return errors.Wrap(err, "error reading config file")
 	}
-
-	return LoadBytes(buf, c)
+	return LoadBytes(buf, expandEnvVars, c)
 }
 
 // LoadBytes unmarshals a config from a buffer. Defaults are not
 // applied to the file and must be done manually if LoadBytes
 // is called directly.
-func LoadBytes(buf []byte, c *Config) error {
+func LoadBytes(buf []byte, expandEnvVars bool, c *Config) error {
+	// (Optionally) expand with environment variables
+	if expandEnvVars {
+		s, err := envsubst.EvalEnv(string(buf))
+		if err != nil {
+			return errors.Wrap(err, "unable to substitute config with environment variables")
```

Comment:
```suggestion
			return fmt.Errorf("unable to substitute config with environment variables: %w", err)
```

Could this use `fmt.Errorf`? There's some legacy uses of `error.Wrap` here but I'm trying to just use Go's builtin wrapping moving forward.